sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuCast.scala

```diff
@@ -560,44 +597,60 @@ object GpuCast extends Arm {
    * Asserts that all values in a column are within the specific range.
    *
    * @param values ColumnVector to be performed with range check
-   * @param minValue Named parameter for function to create Scalar representing range minimum value
-   * @param maxValue Named parameter for function to create Scalar representing range maximum value
+   * @param minValue Range minimum value of input type T
```

Comment:
I remember that these values have been renamed into `minBound` and `maxBound`?

Reply:
Before these values are passing in as scalars, and we rename them to `min/maxBound` in the `withResource` function. But we never changed the argument names in the function or function header.

Reply:
Okay then let's keep these names for consistency :)